FrontEnd/src/components/profileList/ProfileCard.module.css

```diff
@@ -0,0 +1,125 @@
+.company-card {
+  display: flex;
+  width: 1305px;
```

Comment:
Does it correctly work on smaller screens? we support screns up to 1024px and then we make horizontal scroll

Reply:
Removed it at all.